crates/parser/tests/ui/parse-fail/duplicate-functions.wit.result

```diff
@@ -1,4 +1,4 @@
-"foo" defined twice
+Function "foo" already defined as a Function
```

Comment:
I think it might be better to use "function" instead of "Function" here personally (avoiding the capital "F")